app/views/steps/alternatives/court/edit.en.html.erb

```diff
@@ -15,23 +15,30 @@
         <p>If you decide not to go to court you’ll be helping cases involving vulnerable children get to court
           quicker.</p>
       </div>
-      <h2>What happens at court</h2>
+      <h2 class="court">What happens at court</h2>
       <p>A judge will always put your children first. What you want may not be the best thing for your children. You
         must be prepared to accept the judge’s decision, even if you find it difficult.</p>
       <p>The judge’s decision is legally binding. This means that you and the other person must follow the terms of
         the order.</p>
       <p><a href="https://helpwithchildarrangements.service.justice.gov.uk/going-to-court" target="external_link">Find
         out more about going to court </a></p>
-      <h2>Change or enforce an order</h2>
-      <p>The court order isn’t flexible. If you disagree with any part of the order you can ask a court to change it.
-        You can also ask a court to enforce an order if the other person isn’t following it.</p>
-      <p>You’ll need to apply to court again if you want to change or enforce an order.</p>
-      <h2>Representing yourself in court</h2>
+      <h2 class="court">Changing or enforcing an order</h2>
+      <p>The court order isn’t flexible. You’ll need to apply to court again if you want to change it.</p>
+      <p>You or the other person can apply to court to enforce the order if either of you aren't following the terms.</p>
+      <h2 class="court">Representing yourself in court</h2>
       <p>Most people no longer qualify for free legal help. If you don’t qualify, or can’t afford a lawyer, you will
         need to learn
         <a href="https://www.gov.uk/represent-yourself-in-court" target="external_link">how to represent yourself in
           court</a> before you apply.</p>
+      <h2 class="court">Domestic abuse and child abuse</h2>
+      <p>You might be able to get legal aid if you have evidence that you or your children have been victims of abuse.</p>
+      <p>GOV.UK has more information on <a href="https://www.gov.uk/legal-aid/domestic-abuse-or-violence">legal aid and domestic abuse</a>.</p>
+      <p>Find out more about <a href="http://www.refuge.org.uk/get-help-now/help-for-women/recognising-abuse/">domestic abuse</a> and <a href="https://www.nspcc.org.uk/preventing-abuse/child-abuse-and-neglect/">child abuse</a>.</p>
     </div>
+    <div class="govuk-govspeak gv-s-prose">
```

Comment:
I think this extra div block needs to go. It is a duplicate from the below block.